Source/items.h

```diff
@@ -387,27 +387,27 @@ void SetPlrHandGoldCurs(ItemStruct *h);
 void CreatePlrItems(int playerId);
 bool ItemSpaceOk(int i, int j);
 int AllocateItem();
-void GetSuperItemLoc(int x, int y, int *xx, int *yy);
+Point GetSuperItemLoc(Point pos);
 void GetItemAttrs(int i, int idata, int lvl);
 void SaveItemPower(int i, item_effect_type power, int param1, int param2, int minval, int maxval, int multval);
 void GetItemPower(int i, int minlvl, int maxlvl, affix_item_type flgs, bool onlygood);
 void SetupItem(int i);
 int RndItem(int m);
 void SpawnUnique(_unique_items uid, int x, int y);
-void SpawnItem(int m, int x, int y, bool sendmsg);
-void CreateRndItem(int x, int y, bool onlygood, bool sendmsg, bool delta);
-void CreateRndUseful(int x, int y, bool sendmsg);
-void CreateTypeItem(int x, int y, bool onlygood, int itype, int imisc, bool sendmsg, bool delta);
+void SpawnItem(int m, Point pos, bool sendmsg);
+void CreateRndItem(Point pos, bool onlygood, bool sendmsg, bool delta);
+void CreateRndUseful(Point pos, bool sendmsg);
+void CreateTypeItem(Point pos, bool onlygood, int itype, int imisc, bool sendmsg, bool delta);
 void RecreateItem(int ii, int idx, uint16_t icreateinfo, int iseed, int ivalue, bool isHellfire);
 void RecreateEar(int ii, uint16_t ic, int iseed, int Id, int dur, int mdur, int ch, int mch, int ivalue, int ibuff);
 void items_427A72();
 void items_427ABA(Point position);
-void SpawnQuestItem(int itemid, int x, int y, int randarea, int selflag);
+void SpawnQuestItem(int itemid, Point pos, int randarea, int selflag);
 void SpawnRock();
-void SpawnRewardItem(int itemid, int xx, int yy);
-void SpawnMapOfDoom(int xx, int yy);
-void SpawnRuneBomb(int xx, int yy);
-void SpawnTheodore(int xx, int yy);
+void SpawnRewardItem(int itemid, Point pos);
+void SpawnMapOfDoom(Point pos);
+void SpawnRuneBomb(Point pos);
+void SpawnTheodore(Point pos);

```

Comment:
Try avoiding abbreviation, it's not done in other places where Point is being used.